code/missionui/missiondebrief.cpp

```diff
@@ -780,7 +783,9 @@ int debrief_set_stages_and_multi_stuff()
 	}
 
 	for (i=0; i<debriefp->num_stages; i++) {
+		// DISCUSSME: is SEXP_TRUE the only acceptable value? What about SEXP_KNOWN_TRUE?
```

Comment:
`eval_sexp()` will never return `SEXP_KNOWN_TRUE`, so this condition covers both (although it should maybe use `SEXP_TRUE` instead of a magic number 1).

Reply:
ok. so `eval_sexp()` will always return either `SEXP_TRUE` or `SEXP_FALSE`?

I think the other calls to `eval_sexp()` that this PR deals with consider `true` to be any non-zero value (i.e., anything other than `SEXP_FALSE` ). But I guess I should leave the code logic untouched.

So yeah, I guess the only change I'll make is replace `1` with 1SEXP_TRUE`, as you suggested.